### PR TITLE
lots of website redesign, menu too

### PR DESCRIPTION
**Pull request by thepwds**
biig pull request! i added: seperate page support, and a menu that looks nice!
tested and working

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -33,7 +33,7 @@ def get_cover():
     for row in soup.find_all('tr'):
         stats.append(row.get_text())
     print("6")
-    file = stats[9].split(":")[1]
+    file = stats[8].split(":")[1]
```
Comment (summeringpainting) [inline main.py]: Needs to be 9

File: main.py [hunk 2]
```diff
@@ -75,7 +75,7 @@ def getmetadata():
     md = {}
     for row in soup.find_all('tr'):
         stats.append(row.get_text())
-    file = stats[9].split(":")[1]
+    file = stats[8].split(":")[1]
```
Comment (summeringpainting) [inline main.py]: needs to be 9

File: main.py [hunk 3]
```diff
@@ -123,4 +129,4 @@ def Audio_Stream():
 
 
 if __name__ == "__main__":
-    app.run(debug=True, host="127.0.0.1", port=5000)
+    app.run(debug=True, host="10.0.0.195", port=5000)
```
Comment (summeringpainting) [inline main.py]: put host in .env

**Review by summeringpainting — CHANGES_REQUESTED** [review]

File: main.py (hunk 1 above)
Comment (thepwds) [reply main.py]: ok

**Pull request merged**